doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

```diff
@@ -88,18 +93,18 @@ kubectl get pods
 # rayjob-sample-raycluster-9c546-worker-small-group-nfbxm   1/1     Running     0          3m46s
 
 # Step 4.4: Check the status of the RayJob.
-# The field `jobStatus` in the RayJob custom resource will be updated to `SUCCEEDED` once the job finishes.
-kubectl get rayjobs.ray.io rayjob-sample -o json | jq '.status.jobStatus'
+# The field `jobStatus` in the RayJob custom resource will be updated to `SUCCEEDED` and `jobDeploymentStatus`
+# should be `Complete` once the job finishes.
+kubectl get rayjobs.ray.io rayjob-sample -o jsonpath='{.status.jobStatus}'
+# [Expected output]: "SUCCEEDED"
 
-# [Example output]
-# "SUCCEEDED"
+kubectl get rayjobs.ray.io rayjob-sample -o jsonpath='{.status.jobDeploymentStatus}'
+# [Expected output]: "Complete"
 ```
 
-The KubeRay operator will create a RayCluster as defined by the `rayClusterSpec` custom resource, as well as a Kubernetes Job to submit a Ray job to the RayCluster.
-The Ray job is defined in the `entrypoint` field of the RayJob custom resource.
-In this example, the `entrypoint` is `python /home/ray/samples/sample_code.py`,
-and `sample_code.py` is a Python script stored in a Kubernetes ConfigMap mounted to the head Pod of the RayCluster.
-Since the default value of `shutdownAfterJobFinishes` is false, the RayCluster will not be deleted after the job finishes.
+The KubeRay operator creates a RayCluster custom resource based on the `rayClusterSpec` and a submitter Kubernetes Job to submit a Ray job to the RayCluster.
+In this example, the `entrypoint` is `python /home/ray/samples/sample_code.py`, and `sample_code.py` is a Python script stored in a Kubernetes ConfigMap mounted to the head Pod of the RayCluster.
+Since the default value of `shutdownAfterJobFinishes` is false, the KubeRay operator doesn't delete the RayCluster and the submitter once the Ray job finishes.
```

Comment:
```suggestion
Because the default value of `shutdownAfterJobFinishes` is false, the KubeRay operator doesn't delete the RayCluster or the submitter when the Ray job finishes.
```